resources/views/partials/content-page-settings.blade.php

```diff
@@ -1,11 +1,20 @@
-<ul class="link-list">
-  @foreach(get_posts([
-    'post_type' => 'page',
-    'posts_per_page' => -1,
-    'orderby' => 'menu_order',
-    'order' => 'ASC',
-    'post_parent' => $post->ID
-    ]) as $child)
-  <li class="link-list__item"><a href="{{ get_permalink($child->ID) }}">{{ $child->post_title }}</a></li>
-  @endforeach
-</ul>
+<form method="post">
+  <p><strong>{{ __('Allow the Resource Library to track what resources you’ve viewed.', 'coop-library') }}</strong></p>
+  <span class="disclosure-label" hidden>{{ __('How this changes what I see', 'coop-library') }}</span>
```

Comment:
We may want to tweak the copy for this label. The word "this" may not be specific enough.

Perhaps "How resource tracking changes what I see"? @cherylhjli @dayotte  any thoughts?